src/cov/cov-report.c

```diff
@@ -284,51 +282,52 @@ static void cover_print_html_header(FILE *f, cover_report_ctx_t *ctx, bool top,
               "  </head>\n"
               "  <section>\n\n");
 
-   if (!top) {
-      fprintf(f, "<nav>\n"
-                 "<b>Hierarchy:</b><br>\n");
-      int offset = 0;
-
-      ident_t full_hier = s->hier;
-      ident_t curr_id;
-      ident_t curr_hier = NULL;
-      const char *link = "../index";
-      do {
-         curr_id = ident_walk_selected(&full_hier);
-         curr_hier = ident_prefix(curr_hier, curr_id, '.');
-         if (offset > 0)
-            link = istr(curr_hier);
-         if (curr_id)
-            fprintf(f, "<p style=\"margin-left: %dpx\"><a href=%s.html>%s</a></p>\n",
-                        offset * 10, link, istr(curr_id));
-         offset++;
-      } while (curr_id != NULL);
-      fprintf(f, "</nav>\n\n");
-   }
-
    fprintf(f, "<header>");
-   va_start(ap, title);
-   vfprintf(f, title, ap);
-   va_end(ap);
+   fprintf(f, COV_RPT_TITLE "\n");
    fprintf(f, "</header>\n\n");
+}
+
+static void cover_print_navigation_tree(FILE *f, cover_report_ctx_t *ctx, cover_scope_t *s, ...)
```

Comment:
```suggestion
static void cover_print_navigation_tree(FILE *f, cover_report_ctx_t *ctx, cover_scope_t *s)
```

Doesn't need to be a variadic function.